pantheon/src/main/java/tech/pegasys/pantheon/controller/PantheonController.java

```diff
@@ -53,7 +55,16 @@
 
     final GenesisConfigOptions configOptions = genesisConfigFile.getConfigOptions();
 
-    if (configOptions.isEthHash()) {
+    if (FixedDifficultyCalculators.isFixedDifficultyInConfig(configOptions)) {
```

Comment:
is it possible to prevent FixedDifficulty from leaking this far?
I.e. this level just checks ibft vs clique vs ethhash - the ProtocolSchedule is where we check literal config options for the consensus mechanism.
(I say this not knowing how hard this would be to do in the ProtocolSchedule (i also appreciate the mainnetprotocolschedule is a bad name)).

Reply:
Moved to MainnetPantheonController and it feels better there.  Thanks for the feedback.